src/proto/tactic.proto

```diff
@@ -124,8 +124,7 @@ message MoveTactic
     required Point destination = 1;
     // The orientation the robot should have when it arrives at its destination
     required Angle final_orientation = 2;
-    // The speed the robot should have when it arrives at its destination
-    required double final_speed = 3;
+    // 3 reserved field number
```

Comment:
There's an actual `reserved` keyword you can use which will enable the protobuf compiler to generate an error if you try reusing the reserved field number
```suggestion
    reserved 3, 8;
```